samples/pmic/npmx/charger_and_events/README.rst

```diff
@@ -0,0 +1,97 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER AND EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx CHARGER AND EVENTS sample demonstrates how to control PMIC CHARGER and receive events using npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf5340dk_nrf5340_cpuapp
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses CHARGER and EVENTS driver to do the following:
+
+  * Set CHARGER basic configuration.
+  * Register EVENTS from device.
```

Comment:
```suggestion
* Set CHARGER basic configuration.
* Register EVENTS from device.
```

Reply:
What do you mean by CHARGER and EVENTS in this case? SHouldn't they be written in lowercase?